src/flag_gems/ops/bmm.py

```diff
@@ -6,6 +6,15 @@
 
 from ..utils import libentry
 
+# cambricon configs
+blocksmn = [256, 512, 1024]
```

Comment:
Could you please describe what this MR has done?

Reply:
added describe